src/tools.c

```diff
@@ -239,8 +239,9 @@ void tools_execute_custom_command(GeanyDocument *doc, const gchar *command)
 	}
 	else
 	{
-		geany_debug("spawn_sync() failed: %s", error->message);
-		ui_set_statusbar(TRUE, _("Custom command failed: %s"), error->message);
+		ui_set_statusbar(TRUE, _("Cannot execute custom command \"%s\": %s. "
+			"Check the path setting in Preferences."), command, error->message);
```

Comment:
Custom commands are not configured in the Preferences (although they probably should, but that's another topic) but in their own dialog (*Edit → Format → Send Selection to → Set Custom Commands*).